workflows/promote-nightly.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * Promotes a nightly release.
+ */
+
+import {createVersionsUpdatePullRequest, runPromoteJob} from './promote-job';
+
+const jobName = 'promote-nightly.js';
+const {AMP_VERSION} = process.env;
+
+void runPromoteJob(jobName, async () => {
+  if (!AMP_VERSION) {
+    throw new Error('Environment variable AMP_VERSION is missing');
+  }
+
+  await createVersionsUpdatePullRequest(() => ({
+    versionsChanges: {nightly: `04${AMP_VERSION}`},
+    title: `⏫🌙 Promoting release ${AMP_VERSION} to Nightly channel`,
+    body: `Promoting release ${AMP_VERSION} to Nightly channel`,
```

Comment:
The title and body could probably be parameterized. We already have `channel-name` as an input on the workflow - it could be used in `promote-job.ts`

Reply:
We could probably get rid of this file alltogether

Reply:
Not really, different promote workflows will have different `versionMutator` functions (e.g., the opt-in job on Tuesdays will have to take the existing AMP version from Nightly and set it to the opt-in channels). This is the most generic function I could think of atm

Reply:
I was picturing the opt-in workflow to have an additional step - getting the amp version of the current nightly with an optional override - and then calling the promote script.

That would allow the promote script be the same across workflows and I imagine it would be easier to debug if it was split in different steps.

Reply:
Take a look at #36 (still in draft) and lmk if you still think parameterizing this would make it look better (the answer might be yes! You decide)